### PR TITLE
Reorganize to show potential folder structure

### PR DESCRIPTION
**Pull request by laurenmbeatty**
## Description

This is an example of what a component's folder structure could look like. I created a `components` directory. Inside, there is a `Modal` directory, that houses all of the code for the Modal component. One file has all of the styled components for the Modal. Then, there is one main index file in the `components` directory that will export all of the components.

`rosa chopin tried` is a good test token.

## Type of Changes

|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix              |
|   | :sparkles: New feature     |
|  ✓   | :hammer: Refactoring       |
|    | :100: Add tests            |
|    | :link: Update dependencies |
|    | :scroll: Docs              |

## Testing Steps / QA Criteria

`rosa chopin tried` is a good token to use. The delete functionality should work the same as on Karan and Abby's branch.

### CODE REVIEW

File: src/components/Modal/Modal.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import { Modal as MuiModal } from '@material-ui/core';
+import React from 'react';
+import { StyledModalBtn, StyledModalLayout } from './elements';
+
+const Modal = ({
```
Comment (laurenmbeatty) [inline src/components/Modal/Modal.js:5]: This is now a semi-generic Modal component that could be used in a number of situations.

File: src/components/Modal/elements.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
+import styled from 'styled-components';
+
+// These elements are really just associated with the DeleteModal component
+// No need to make these available everywhere
+// You could have a more Global elements file, as well,
+// as well as use the `GlobalStyles` component from styled-components
+export const StyledModalBtn = styled.button`
```
Comment (laurenmbeatty) [inline src/components/Modal/elements.js:7]: It's nice for a component to have its own elements file, if necessary.

File: src/Item.js [hunk 3]
```diff
@@ -123,23 +122,19 @@ function Item({ userToken, item }) {
       </button>
       {/* // [delete-item] 2. Create a button modal that will render when user tries to delete item */}
       <Modal
-        open={openModal}
+        cancelLabel="Cancel"
+        confirmLabel="Confirm"
```
Comment (laurenmbeatty) [inline src/Item.js:126]:
```suggestion
        confirmLabel="Yes"
```

File: src/components/index.js [hunk 4]
```diff
@@ -0,0 +1,4 @@
+import Modal from './Modal';
+
+// Then you can export all of the components from here
+export { Modal };
```
Comment (kkatoch89) [inline src/components/index.js:4]: I really like this structure. Team Cantimore, can we organize it this way?

**Review by kkatoch89 — APPROVED** [review]

**Review by cafloyd — APPROVED** [review]

**Pull request merged**